projects/step-frontend/src/lib/os-plugins/plugin.module.ts

```diff
@@ -9,10 +13,10 @@ import { StepCoreModule, PluginLazyLoad, ImportMeta } from '@exense/step-core';
 export class PluginModule extends PluginLazyLoad {
```

Comment:
Why do we need export const registerOsPlugins = async (injector: Injector): Promise<void> => {... between this module and plugin-initializer? As we don't do any dynamic plugin loading in OS the code would be simpler to directly call this in the plugin-initializer.

Reply:
@neogucky dynamic import for PluginModule in OS has been removed